# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fs/test/gtest/FileSystemMocks.h
# Commit: e4b3f3d8ba5f
# Full Hash: e4b3f3d8ba5fd951bf79a2014dcee109255c865f
# Author: Jan Varga <jvarga@mozilla.com>
# Date: 2022-08-31 21:54:47
# Regressor Bug: 1786465
# File Overlap Count: 1
# Description:
#   Bug 1786465 - Rename POriginPrivateFileSystem to PFileSystemManager; r=dom-storage-reviewers,jesup
#   
#   The child actor will be held by FileSystemManager on the child side and the
#   parent actor already holds FileSystemDataManager, so it makes sense to rename
#   the protocol to PFileSystemManager.
# ==============================================================================

diff -r 3b0da4b4bb83 -r e4b3f3d8ba5f dom/fs/test/gtest/FileSystemMocks.h
--- a/dom/fs/test/gtest/FileSystemMocks.h	Wed Aug 31 09:34:48 2022 +0000
+++ b/dom/fs/test/gtest/FileSystemMocks.h	Wed Aug 31 09:34:48 2022 +0000
@@ -19,7 +19,7 @@
 #include "mozilla/dom/BindingUtils.h"
 #include "mozilla/dom/DOMException.h"
 #include "mozilla/dom/DOMExceptionBinding.h"
-#include "mozilla/dom/OriginPrivateFileSystemChild.h"
+#include "mozilla/dom/FileSystemManagerChild.h"
 #include "mozilla/dom/Promise.h"
 #include "mozilla/dom/PromiseNativeHandler.h"
 #include "mozilla/dom/ScriptSettings.h"
@@ -193,7 +193,7 @@
   ErrorHandler mOnError;
 };
 
-class TestOriginPrivateFileSystemChild : public OriginPrivateFileSystemChild {
+class TestFileSystemManagerChild : public FileSystemManagerChild {
  public:
   MOCK_METHOD(void, SendGetRootHandle,
               (mozilla::ipc::ResolveCallback<FileSystemGetHandleResponse> &&
@@ -246,22 +246,22 @@
   MOCK_METHOD(void, Shutdown, (), (override));
 
  protected:
-  virtual ~TestOriginPrivateFileSystemChild() = default;
+  virtual ~TestFileSystemManagerChild() = default;
 };
 
 class TestFileSystemChildFactory final : public FileSystemChildFactory {
  public:
-  explicit TestFileSystemChildFactory(TestOriginPrivateFileSystemChild* aChild)
+  explicit TestFileSystemChildFactory(TestFileSystemManagerChild* aChild)
       : mChild(aChild) {}
 
-  already_AddRefed<OriginPrivateFileSystemChild> Create() const override {
-    return RefPtr<TestOriginPrivateFileSystemChild>(mChild).forget();
+  already_AddRefed<FileSystemManagerChild> Create() const override {
+    return RefPtr<TestFileSystemManagerChild>(mChild).forget();
   }
 
   ~TestFileSystemChildFactory() = default;
 
  private:
-  TestOriginPrivateFileSystemChild* mChild;
+  TestFileSystemManagerChild* mChild;
 };
 
 struct MockExpectMe {